Gruntfile.js

```diff
@@ -55,7 +62,8 @@ module.exports = function(grunt) {
 
   // Whenever the "test" task is run, first clean the "tmp" dir, then run this
   // plugin's task(s), then test the result.
-  grunt.registerTask('test', ['clean', 'chimp', 'nodeunit']);
+  grunt.registerTask('test', ['clean', 'chimp:cucumber', 'nodeunit']);
```

Comment:
aha, i see what you mean. we could certainly do this. it would be configured in `audi-grunt`. once we update this repo I'll work on that.